src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.card {
+  &__stars {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    line-height: 14px;
+    padding: $product-img-padding 0;
+
+    &-reviews {
+      margin: 0;
+      font-size: $font-S;
+      color: $title-color;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 4px;
+  width: 96px;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 .stars__star:nth-last-child(1n + 2) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
The way you've implemented the stars is not very flexible. If you want to change the number of active stars, you have to add a new class and rule for every possible number. Consider a more flexible solution, for example, you could use a loop to generate the classes and rules for all possible numbers of active stars.